packages/Webkul/Shop/src/Routes/api.php

```diff
@@ -22,9 +22,17 @@
             ->name('shop.categories.maximum-price');
     });
 
+    Route::controller(CartController::class)->group(function () {
+        Route::post('cart', 'store')->name('shop.checkout.cart.store');
+
+        Route::prefix('checkout/cart')->group(function () {
+            Route::get('', 'index')->name('shop.checkout.cart');
+        
+            Route::delete('remove/{id}', 'destroy')->name('shop.checkout.cart.destroy');
+        });
```

Comment:
Changes are not there which we have discussed already.

Reply:
Added checkout prefix in all 3 routes